internal/monitors/collectd/consul/consul.go

```diff
@@ -29,33 +33,73 @@ const monitorType = "collectd/consul"
 func init() {
 	monitors.Register(monitorType, func() interface{} {
 		return &Monitor{
-			*collectd.NewMonitorCore(CollectdTemplate),
+			python.PyMonitor{
+				MonitorCore: pyrunner.New("sfxcollectd"),
+			},
 		}
 	}, &Config{})
 }
 
 // Config is the monitor-specific config with the generic config embedded
 type Config struct {
 	config.MonitorConfig `yaml:",inline" acceptsEndpoints:"true"`
+	pyConf               *python.Config
+	Host                 string `yaml:"host" validate:"required"`
+	Port                 uint16 `yaml:"port" validate:"required"`
+	ACLToken             string `yaml:"aclToken" neverLog:"true"`
+	UseHTTPS             bool   `yaml:"useHTTPS"`
+	EnhancedMetrics      bool   `yaml:"enhancedMetrics"`
+	CACertificate        string `yaml:"caCertificate"`
+	ClientCertificate    string `yaml:"clientCertificate"`
+	ClientKey            string `yaml:"clientKey"`
+	SignalFxAccessToken  string `yaml:"signalFxAccessToken" neverLog:"true"`
+}
 
-	Host string `yaml:"host" validate:"required"`
-	Port uint16 `yaml:"port" validate:"required"`
-
-	ACLToken            string `yaml:"aclToken" neverLog:"true"`
-	UseHTTPS            bool   `yaml:"useHTTPS"`
-	EnhancedMetrics     bool   `yaml:"enhancedMetrics"`
-	CACertificate       string `yaml:"caCertificate"`
-	ClientCertificate   string `yaml:"clientCertificate"`
-	ClientKey           string `yaml:"clientKey"`
-	SignalFxAccessToken string `yaml:"signalFxAccessToken" neverLog:"true"`
+// PythonConfig returns the embedded python.Config struct from the interface
+func (c *Config) PythonConfig() *python.Config {
+	return c.pyConf
 }
 
 // Monitor is the main type that represents the monitor
 type Monitor struct {
-	collectd.MonitorCore
+	python.PyMonitor
 }
 
 // Configure configures and runs the plugin in collectd
-func (am *Monitor) Configure(conf *Config) error {
-	return am.SetConfigurationAndRun(conf)
+func (m *Monitor) Configure(conf *Config) error {
+	conf.pyConf = &python.Config{
+		MonitorConfig: conf.MonitorConfig,
+		Host:          conf.Host,
+		Port:          conf.Port,
+		ModuleName:    "consul_plugin",
+		ModulePaths:   []string{filepath.Join(os.Getenv(constants.BundleDirEnvVar), "plugins", "collectd", "consul")},
```

Comment:
This filepath construction gets repeated a lot.  Can you make some kind of function, in some appropriate place (not the utils package), that can be called something like `makeCollectdPath("consul")` so that this is shorter and more centralized as far as where this directory actually lives.